project.php

```diff
@@ -1,7 +1,7 @@
 <?php
 $relPath = './pinc/';
 include_once($relPath.'base.inc');
-include_once($relPath.'stages.inc');
+include_once($relPath.'stages.inc'); // validate_user_can_get_pages_in_project()
```

Comment:
We use a lot of other stuff in `stages.inc` in this file, I would leave off the comment about this one function lest others think this is the only thing we use in that file. Yeah, we're wildly inconsistent about how we do this 😕